### ISSUE TITLE
Support captions

### ISSUE THREAD

**Issue by jjchambl** [madeagency/xlr#6]
While the readme contains captions for the column headings, inserting these into the spreadsheet does not generate them as expected. Are these actually supported?

**Comment by fjlaubscher** [comment]
Hey @jjchambl 

The first / early version of XLR did support _captions_ but it was more column headers, if anything. (The original library XLR was based off, called it captions)
By setting a column's caption, it would just automatically add a column on the first row with the column header.

It has since been removed to support cell merging and styling and you'd have to add a column header manually by doing something like the following.
```[{ value: 'First Name', style: 1 }, { value: 'Last Name', style: 1 }, { value: 'Email', style: 1 }]```

I just should've updated the example in the readme file  -- sorry about that 😅

If you feel like you'd be able to achieve automatically generated column headers while maintaining cell styling and cell merging, feel free to open a PR 🚀

**Comment by jjchambl** [comment]
Yeah I found that out yesterday, just got confused by the readme. I wrote something that adds in the column headers yesterday, but I'll need to clean it up before making a pr. May be able to get to that in a week or two.
